modelzoo/utils.py

```diff
@@ -0,0 +1,66 @@
+import typing
+import base64
+import io
+import os
+from w3lib.url import parse_data_uri
+import mimetypes
+from PIL import Image
+from pathlib import Path
+from _protos.services_pb2 import (
+    TextGenerationRequest,
+    ImageSegmentationRequest,
+    VisionClassificationRequest,
+    ModelUUIDRequest,
+    ModelUUIDResponse
+)
+
+class UnauthorizedException(Exception):
+    pass
+
+ImgLike = typing.NewType('ImgLike', typing.Union[type(Image), str])
+
+def img_file_to_uri(filepath: str) -> str:
+    """
+        Reads an JPEG image from a file and converts it to a data URI.
+    """
+    if not os.path.isfile(filepath):
+        raise FileNotFoundError()
+    img = Image.open(filepath)
+    return img_to_uri(img)
+
+def img_to_uri(img: Image) -> str :
+    """
+        Converts a JPEG Image (In the form of a PIL.Image) to a data URI.
+    """
+    buffered = io.BytesIO()
+    img.save(buffered, format="JPEG")
+    img_str = base64.b64encode(buffered.getvalue()).decode()
+    return u'data:%s;base64,%s' % (mimetypes.types_map['.jpg'], img_str)
```

Comment:
use `mimetypes.guess_type` instead of hardcoding it. https://gist.github.com/jsocol/1089733